src/components/MessagesList/MessagesList.vue

```diff
@@ -1221,9 +1238,14 @@ export default {
 			}
 		},
 
-		setChatScrolledToBottom(boolean) {
-			this.$emit('update:is-chat-scrolled-to-bottom', boolean)
-			if (boolean) {
+		setChatScrolledToBottom({ atBottom = true, auto = false } = {}) {
```

Comment:
Since we're always passing the first argument (by the definition of this method), maybe it's better to use this notation?
```suggestion
		setChatScrolledToBottom(value, { auto = false } = {}) {
```